src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -384,51 +382,131 @@ void getHearingLocations() {
             .dwpIsOfficerAttending("Yes")
             .build();
 
-        given(venueService.getEpimsIdForVenue(caseData.getProcessingVenue())).willReturn("219164");
+        given(venueService.getEpimsIdForVenue(caseData.getProcessingVenue())).willReturn(EPIMS_ID_1);
         given(referenceDataServiceHolder.getVenueService()).willReturn(venueService);
 
-        List<HearingLocation> result = HearingsDetailsMapping.getHearingLocations(caseData, referenceDataServiceHolder);
-
-        assertThat(result)
-            .hasSize(1)
-            .extracting("locationId","locationType")
-            .containsExactlyInAnyOrder(tuple("219164", COURT));
+        checkHearingLocationResults(HearingsLocationMapping.getHearingLocations(caseData, referenceDataServiceHolder),
+                                    EPIMS_ID_1);
     }
 
     @DisplayName("When override Hearing Venue Epims Ids is not empty getHearingLocations returns the override values")
     @Test
-    void getHearingLocationsOverride() {
+    void getHearingLocationsOverride() throws InvalidMappingException {
+        buildOverrideHearingLocations();
 
-        SscsCaseData caseData = SscsCaseData.builder()
-            .appeal(Appeal.builder()
-                .hearingOptions(HearingOptions.builder().build())
-                .build())
-            .schedulingAndListingFields(SchedulingAndListingFields.builder()
-                .overrideFields(OverrideFields.builder()
-                    .hearingVenueEpimsIds(List.of(
-                        CcdValue.<CcdValue<String>>builder()
-                            .value(CcdValue.<String>builder()
-                                .value("219164")
-                                .build())
-                            .build(),
-                        CcdValue.<CcdValue<String>>builder()
-                            .value(CcdValue.<String>builder()
-                                .value("436578")
-                                .build())
-                            .build()))
-                    .build())
-                .build())
-            .processingVenue(PROCESSING_VENUE_1)
-            .build();
+        checkHearingLocationResults(HearingsLocationMapping.getHearingLocations(caseData, referenceDataServiceHolder),
+                                    EPIMS_ID_1, EPIMS_ID_2);
+    }
+
+    @DisplayName("When a case has been adjourned and a different venue has been selected, return the new venue")
```

Comment:
Can we add a clause to this test (and perhaps another separate test) to show that 
* changing venue to a different RPC saves the new RPC
* changing venue to the same RPC saves the same RPC

Reply:
Added it so it will show that the RPC gets set. I've only done this once as those tests are more testing the correct functionality of the venue services rather than my code